src/objective-c/GRPCClient/GRPCCall.m

```diff
@@ -92,7 +93,7 @@ @interface GRPCCall () <GRXWriteable>
 @implementation GRPCCall {
   dispatch_queue_t _callQueue;
```

Comment:
Thanks. The comment is still confusing because it's explaining implementation choices based on things that don't happen in this class anymore. But it doesn't work to move it into GRPCWrappedCall as is: We'll have to move _callQueue and _callAlreadyInvoked there, making GRPCWrappedCall thread-safe. Let's do it in a separate PR though, to keep this one focused.